### DISCUSSION TITLE
Main

### DISCUSSION THREAD

**Discussion by rafaelg7** [9eu1/France-Project.github.io#1] [General]
@flxp229 
Hast du noch eine Idee was ich bei Crepes machen kann?

**Comment by ghost** [comment]
@rafaelg7 

Ganz oben auf der Seite würde ich, ähnlich wie bei der WFC-eSports Homepage, ein fettes Bild hinplatzieren mit der Schrift "Crêpes" drauf.

Unten drunter schreiben wir dan was dazu, da kannst du aber einfach schonmal Plazhaltertext (lorem ipsum) hinschreiben, das passen wir an, wenn's soweit ist. Ganz unten würde ich vlt noch eine kleine Bildergalerie (Die Bilder schießt Tim) einfügen, um das ganze schön abzuschließen. Irgendwo kommt dann halt noch  die Adresse zum Bestellen der Crêpes hin, aber mach dir darum mal zunächst keine Gedanken.

Auf jede  "unterseite" kommt unten noch ein kleiner "footer" (siehe WFC-Homepage") hin, der überall einheitlich ist. Wenn du willst, kannst du schonmal einen erstellen.

**Reply by rafaelg7** [reply]
Mach ich wahrscheinlich erst morgen oder Sonntag. da ich heute Abend noch was vor habe.

**Reply by ghost** [reply]
Passt schon, eilt ja nicht.

**Comment by ghost** [comment]
Hey Rafael, möchtest du lieber die Crêpes- oder die Projektseite machen? (Um CSS kann ich mich kümmern)

**Reply by rafaelg7** [reply]
Ich mach die Crêpesseite.

**Comment by rafaelg7** [comment]
Wenn ich am Donnerstag nicht fertig werde, kannst du noch bitte weiter arbeiten?
